tests/v2/validation/nodescaling/scaling_nodepools.go

```diff
@@ -27,6 +27,10 @@ func scalingRKE2K3SNodePools(t *testing.T, client *rancher.Client, clusterID str
 	cluster, err := client.Steve.SteveType(ProvisioningSteveResourceType).ByID(clusterID)
 	require.NoError(t, err)
 
+	if nodeRoles.Windows {
```

Comment:
I think you mentioned this offline but why is this needed for windows but not other node types?

Reply:
Yeah, Windows needs to specifically have the syntax shown in the if condition. If I try to do the scale down syntax that Linux nodes do, it doesn't work. Vice versa, it's the same story.